helix-core/src/main/java/org/apache/helix/model/ClusterConfig.java

```diff
@@ -822,6 +840,19 @@ public void setGlobalRebalancePreference(Map<GlobalRebalancePreferenceKey, Integ
     return DEFAULT_GLOBAL_REBALANCE_PREFERENCE;
   }
 
+  /**
+   * Set the asynchronous global rebalance mode.
+   * @param isAsync true if the global rebalance should be performed asynchronously
+   */
+  public void setGlobalRebalanceAsyncMode(boolean isAsync) {
```

Comment:
Nit: `enableGlobalRebalanceAsyncMode`?

Reply:
Please see my reply in #632. enableXXXX is not a preferred name.
Considering enableXXXX(false), it is very easy to get lost if you are enabling it or disabling it.

setXXXX is more straight forward.